test/environments/norails/Gemfile

```diff
@@ -4,20 +4,10 @@ gem 'rake', '~> 12.3.3'
 
 gem 'minitest', '~>4.7.5'
 gem 'mocha', '~> 1.9.0', :require => false
+gem 'rack', '>= 2.1.4'
+gem 'rack-test', '< 0.8.0'
```

Comment:
For rack and rack-test, how did you decide on these versions? it seems like 0.8.0 was used on  ruby < 2.2, looking at the diff. just curious!

Reply:
These versions came from [dependabot alerts](https://github.com/newrelic/newrelic-ruby-agent/security/dependabot) raised about versions below 2.1.4. I upgraded the version where I could to avoid the vulnerability.

Reply:
And no worries! Good question!

Reply:
ahh ok cool, thank makes sense, thank you for explaining!